pkgs/tools/networking/grpc_cli/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, automake, cmake, autoconf, curl, numactl }:
+
+stdenv.mkDerivation rec {
+  pname = "grpc_cli";
+  version = "1.46.6";
+
+  src = fetchFromGitHub {
+    owner = "grpc";
+    repo = "grpc";
+    rev = "v${version}";
+    hash = "sha256-m55FKwqlAQ2Ge7y4RMFO4W1onu0NYARWrOygChJMxng=";
+    fetchSubmodules = true;
+  };
+  buildInputs = [ automake cmake autoconf curl numactl ];
+  configurePhase = "true";
+  buildPhase = ''
+    mkdir -p $out/bin
+    mkdir -p cmake/build
+    cd cmake/build
+    cmake -DgRPC_BUILD_TESTS=ON ../..
+    make grpc_cli
+  '';
```

```suggestion
  makeFlags = [ "grpc_cli" ];
```

Also remove the configurePhase override above.

---

I also added cmakeFlags and it worked. Thank you!